spec/std/float_spec.cr

```diff
@@ -206,11 +206,13 @@ describe "Float" do
 
   describe "hash" do
     it "does for Float32" do
-      1.2_f32.hash.should_not eq(0)
+      1.2_f32.hash.should eq(1.2_f32.hash)
+      1.2_f32.hash.should_not eq(1.3_f32.hash)
```

Comment:
this expectation is wrong in common case.

Reply:
I'll remove this then. Thanks